internal/stanza/converter.go

```diff
@@ -560,7 +551,18 @@ func getResourceID(resource map[string]string) uint64 {
 		fnvHash.Write([]byte(k))
 		fnvHash.Write(pairSep)
 
-		fnvHash.Write([]byte(resource[k]))
+		switch t := resource[k].(type) {
+		case string:
+			fnvHash.Write([]byte(t))
+		case []byte:
+			fnvHash.Write(t)
+		case int, int64:
+			binary.Write(fnvHash, binary.BigEndian, t)
+		default:
```

Comment:
Can float be also one of the types?

Reply:
Yeah, good call. I should really support most/all the same types as the conversion code already names. I'll update.